activity-embedding-advanced/README.md

```diff
@@ -0,0 +1,35 @@
+# Activity Embedding
+
+The activity embedding sample app is the codebase of advanced activity
+embedding codelab.
+
+The codelab guides you step by step through the process of implementing 
+activity embedding features:
+
+* Pane expansion (interactive divider): allows apps to display a fixed or 
+draggable divider between the two activities in a split presentation.
+* ActivityStack Pinning: allows apps to pin the content in one container 
+and have its navigation isolated from the other container.
+* Fullscreen Dialog Dimming: allows apps to specify the dialog dim area,
```

Comment:
Full-screen dimming for dialogs:

This gets around the problem of what "full-screen" modifies.